Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gemspec
+# gemspec # This causes my `bundle install` to fail
```

Comment:
http://yehudakatz.com/2010/12/16/clarifying-the-roles-of-the-gemspec-and-gemfile/ says to use gemspec